docs/theming.rst

```diff
@@ -163,6 +168,13 @@ The following keys are currently supported:
   * ``ignored_assets`` — comma-separated list of assets to ignore (relative to
     the ``assets/`` directory, eg. ``css/theme.css``)
 
+* ``jinja`` - This section is ignored unless your theme's engine is ``jinja``.
+
+  * ``extensions`` - comma-separated list of
```

Comment:
I’m OK with this, but perhaps it would be more useful to do this in `conf.py` instead?

Reply:
I first thought the opportune place for template configuration is the theme configuration file. It is there I tell Nikola which engine to use in the first place.

On the other hand, `conf.py` would allow real python code, as opposed to mere strings. It also has the global context and other such goodies, they may interact with the template. Since presenting this PR, I've also encountered the desire to have my templates [bark at undefined varialbes](https://jinja.palletsprojects.com/en/3.1.x/api/#jinja2.StrictUndefined) as opposed to just printing them as an empty string.

So I'll agree doing something like this in `conf.py` is the better idea, and doing it in a way that allows any configuration change (via Python), as opposed to just allowing extension activation and nothing else.